### PR TITLE
Added onto the README.md 

### PR DESCRIPTION
**Pull request by LukeTheEngineer**
No changes to the source code, only to the README.md section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ See build(7), config(8), [FreeBSD handbook on building userland](https://docs.fr
 For information on the CPU architectures and platforms supported by FreeBSD, see the [FreeBSD
 website's Platforms page](https://www.freebsd.org/platforms/).
 
+Want to snag an ISO of FreeBSD? Head to the officials [releases page](https://download.freebsd.org/ftp/releases/ISO-IMAGES/) to get yourself one. 
+
```
Comment (bsdimp) [inline README.md:19]: Love the information. 
I'd suggest a little more formal phrasing:

```
Official FreeBSD bootable images on our [releases page](...).
```

Comment (bsdimp) [reply README.md:19]: Iso is becoming a bit of an anachronism

Comment (LukeTheEngineer) [reply README.md:19]: Made the change. How does it look?

**Comment by bsdimp** [comment]
Why close?

**Comment by LukeTheEngineer** [comment]
> Why close?

I assumed it wouldn't be merged

**Pull request closed without merging**